src/Notes/WC_Admin_Notes_Onboarding_Email_Marketing.php

```diff
@@ -43,6 +40,6 @@ public static function possibly_add_onboarding_email_marketing_note() {
 		$note->set_name( self::NOTE_NAME );
 		$note->set_source( 'woocommerce-admin' );
 		$note->add_action( 'yes-please', __( 'Yes please!', 'woocommerce-admin' ), 'https://woocommerce.us8.list-manage.com/subscribe/post?u=2c1434dc56f9506bf3c3ecd21&amp;id=13860df971&amp;SIGNUPPAGE=plugin' );
-		$note->save();
+		return $note;
```

Comment:
Is the note type meant to be set to `E_WC_ADMIN_NOTE_MARKETING` here? I can't get any of the generated notes to go away when clearing the marketplace suggestions, the retrieved notes in `possibly_delete_marketing_notes` is empty.

Reply:
@becdetat Based on @pmcpinto's feedback above, the only (for now) notes remaining that are marketing notes are `WC_Admin_Notes_Draw_Attention` and `WC_Admin_Notes_WooCommerce_Payments`.

But I'm guessing based on the above feedback that those notes may be reclassified as well shortly in which case we won't have any "marketing" notes to test against.

If this is the case, we'll have to manually test by adding `WC_Admin_Note::E_WC_ADMIN_NOTE_MARKETING` to notes and attempting to trigger them.

Reply:
Ok thanks